scripts/mage2vs.js

```diff
@@ -36,6 +36,7 @@ function getMagentoDefaultConfig(storeCode) {
     MAGENTO_ACCESS_TOKEN: apiConfig.accessToken,
     MAGENTO_ACCESS_TOKEN_SECRET: apiConfig.accessTokenSecret,
     MAGENTO_URL: apiConfig.url,
+    MAGENTO_MSI_STOCK_ID: 1,
```

Comment:
It should be rather: config.msi.defaultStockId

Reply:
@pkarw Fixed.